src/components/FeedInfo.tsx

```diff
@@ -243,9 +243,13 @@ export class FeedInfo extends React.Component<Props, FeedInfoState> {
             Debug.log('fetchFeedFromUrl', 'url', url);
             const canonicalUrl = Utils.getCanonicalUrl(url);
             Debug.log('fetchFeedFromUrl', 'canonicalUrl', canonicalUrl);
-            const feed = await this.fetchRSSFeedFromUrl(canonicalUrl);
-            Debug.log('fetchFeedFromUrl', 'feed', feed);
-            return feed;
+            try {
```

Comment:
This `try` breaks the build and it's not necessary, because there is a try in `fetchRSSFeedFromUrl` already.

Reply:
yes, this was not planned to be in the PR, it was just for debugging